### ISSUE TITLE
.sandstorm/sandstorm-pkgdef.capnp should probably exclude .git

### ISSUE THREAD

**Issue by zarvox** [JamborJan/paperwork#12]
Hey JJ,

It took me a while to notice this, but the reason that you're hitting the vboxsf file descriptor limits is because you're including a bit more in your .spk than what paperwork actually appears to use.

In particular, you're specifying `alwaysInclude` for `opt/app` and `usr/lib/python3.4`, which I suspect you might not actually need to run paperwork.  (Do correct me if I'm wrong!)  In addition to a bunch of your dependencies' tests, you're probably pulling in a full copy of `.git/` to boot, and any .spk files you have lying around in that folder as well.

I note that the sandstorm-files.list only names a meager 884 files; consider removing the entries in `alwaysInclude` and you might not have to use the NFS shared folder workaround.

**Comment by zarvox** [comment]
Oh, some numbers: I get an 18MB .spk without the alwaysInclude line, and a 61MB one with it as-is.

A significant portion of that is the ~32MB or so of `node_modules` that are only needed at build time, and not runtime.

**Comment by JamborJan** [comment]
You are right, FYI the reason it was in there before:

The ```alwaysInclude```has been added back when I used the build process via Docker. There I had this in the sandstorm logs when starting a grain:

```
Could not find platform independent libraries <prefix>
Consider setting $PYTHONHOME to <prefix>[:<exec_prefix>]
Fatal Python error: Py_Initialize: Unable to get the locale encoding
```
Back then I checked out the Lychee port: https://github.com/jparyani/Lychee/blob/sandstorm/sandstorm-pkgdef.capnp#L54. 

I added "usr/lib/python3.4" to `alwaysInclude`, and it was fixed.

I removed this now and have the same size as you mentioned in your test and the app is still working so it has been depreciated stuff from the Docker build process.

Thanks!
